README.md

```diff
@@ -1515,6 +1515,265 @@ __NOTE__: This section does not go over any additional tuning that can be done
 for the kernel datapath.
 
 
+## Test with Open vSwitch full offload (data and control path) to NIC
+
+
+Offloading OVS entirely to a NIC requires the NIC to support execution of the control plane in addition to accelerating the data plane of OVS. We'll take the example of Cavium's LiquidIO-II SmartNIC
+adapter for this example.
+
+Follow the documentation from [here](#DUTsetup) for the initial setup.
+
+Since the control and data plane functions of OVS are offloaded to the NIC, the openvswitch service
+must be stopped in the host server (DUT).
+
+```
+# systemctl disable openvswitch
+# systemctl stop openvswitch
+# rmmod openvswitch
+```
+
+To allow full offload of OVS to LiquidIO-II adapter, the liquidio driver kernel module must be configured
+to use the appropriate firmware
+
+```
+modprobe liquidio fw_type=vsw
+```
+
+Now, let's enable the SRIOV Virtual functions for the LiquidIO-II adapter. For this example, we'll enable 1 SRIOV VF
+for Physical Function 0
+
+```
+# lspci -d 177d:9702
+
+05:00.0 Ethernet controller: Cavium, Inc. CN23XX [LiquidIO II] Intelligent Adapter (rev 02)
+05:00.1 Ethernet controller: Cavium, Inc. CN23XX [LiquidIO II] Intelligent Adapter (rev 02)
+
+# echo 1 > /sys/bus/pci/devices/0000\:05\:00.0/sriov_numvfs
+```
+
+This will enable 1 VF for PF0 which will be attached to Qemu as a PCI device
+
+```
+# lspci -d 177d:9712
+
+05:00.3 Ethernet controller: Cavium, Inc. CN23XX [LiquidIO II] SRIOV Virtual Function (rev 02)
+```
+
+### Establish communication between DUT and OVS Control Plane on NIC
```

Comment:
Establish communication between __the__ DUT and OVS Control Plane on __the__ NIC